.eslintrc.js

```diff
@@ -0,0 +1,21 @@
+module.exports = {
+  extends: 'kartotherian',
+  env: {
+    node: true,
+    es6: true,
+    browser: false,
+  },
+  rules: {
+    'comma-dangle': [
+      'error',
+      {
+        arrays: 'always-multiline',
+        objects: 'always-multiline',
+        imports: 'always-multiline',
+        exports: 'always-multiline',
+        // Dangling commas are unsupported in node
+        functions: 'never',
+      },
+    ],
+  },
```

Comment:
are these needed? I thought I fixed and added all of them to the main style

Reply:
They don't work otherwise; without the explicit override, grunt is complaining about comma dangles it shouldn't:
```
core/lib/core.js
  283:40  error  Missing trailing comma  comma-dangle
  330:29  error  Missing trailing comma  comma-dangle

core/lib/sources.js
  103:54  error  Missing trailing comma  comma-dangle
  446:17  error  Missing trailing comma  comma-dangle
```